pulp_rpm/app/models/package.py

```diff
@@ -13,9 +15,43 @@
     PULP_PACKAGE_ATTRS,
 )
 
+
 log = getLogger(__name__)
 
 
+# Hard to move this due to circular import problems
+class RpmVersionField(models.Field):
+    """Model Field for evr_t, a custom type for representing RPM EVR."""
+
+    def db_type(self, connection):
+        """Returns the database column type."""
+        return 'evr_t'
+
+
+class PackageManager(models.Manager):
+    """Custom Package object manager."""
+
+    def with_age(self):
+        """Provide an "age" score for each Package object in the queryset.
+
+        Annotate the Package objects with an "age". Age is calculated with a postgresql
+        window function which partitions the packages table by name and architecture,
+        orders the packages in each group by 'evr', and returns the row number of each
+        package, which is the relative "age" within the group. The newest package gets
+        age=1, second newest age=2, and so on.
+
+        Partitioning by architecture is important because there i686 and x86_64 packages
+        with same name and same or close verison numbers are not interchangeable.
```

Comment:
It just crossed my mind that in Pulp3, SRPMs are also stored as Package, the only difference is in arch. 
Since you are already taking care of it, maybe it make sense to throw in a comment about SRPMs specifically, it might not be obvious that they are covered.